CIP-066/README.md

```diff
@@ -0,0 +1,426 @@
+---
+CIP: 066
+Title: NFT Identity
+Authors: Patrick Tobler <patrick.tobler@nmkr.io>, Dennis Mittmann <dennis@iamx.id>
+Status: Active
+Type: Process
+Created: 2022-07-06
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP defines a standard to verify NFT projects using Decentralized Identifiers.
+
+## Motivation
+
+The verification process of the authenticity of NFT projects is currently a centralized effort, that is completely in the hands of the NFT platforms. 
+
+Most of these platforms create a centralized database of verified projects and have arbitrary rules that define which NFTs get verified on the respective platform. Not only is this not scalable because it relies on a case-to-case verification process often done manually and requires the NFT creators to go through the verification process on an increasing amount of different platforms, but it is also highly centralized and doesn't utilize a global cross-platform verification method.
+
+## High Level Overview
+
+By connecting an NFT collection to one or multiple Digital Identifiers, the creator of the NFT collection has the option to attach verification data points to the collection. 
+
+Each of these data points stores information about a direct connection between the NFT collection and some sort of identity medium. This identity medium could be a scoial account, a KYC process, a phone number or anything else where an identity is directly linked to an account or something similar.
+
+The data points connected to the NFT collection are publicly visible and can be audited by anyone. Either manually or programmatically.
+
+The creator can choose which identity data points he wants to connect to the collection and the auditor can then choose if he trusts these data points enough to interact with the colleciton or not.
+
+## Practical Example
+
+Let's imagine Paris Hilton wants to release an NFT collection on Cardano.
```

Comment:
Use of the particular celebrity Paris Hilton doesn't add anything to the substance of the CIP.  If the "story" were a bit shorter and maybe used a more generic term "celebrity" then there would be room in the reader's attention for at least one other use case: that of the unknown artist who's less followed on social media & has less likelihood of being authenticated through a conventional platform.

Reply:
Agreed. Alternatively, could pick a nameless name like _John Doe_ / _Jane Doe_ for the sake of the example.